docs/test.md

```diff
@@ -0,0 +1,98 @@
+# Test
+
+### Test a Dataset
```

Comment:
should be second level header?

Reply:
Will there be another section on "test an image"?

Reply:
> Will there be another section on "test an image"?

Yes, but it will be added to the inference section.